src/features/auth/pages/Login.jsx

```diff
@@ -1,7 +1,93 @@
 import React from "react";
+import { Box, FormHelperText, Typography, Card, CardContent, Grid } from '@mui/material'
+import { useForm } from 'react-hook-form'
+import { yupResolver } from '@hookform/resolvers/yup'
+import * as yup from "yup";
+import {useNavigate} from 'react-router-dom'
+import {useDispatch, useSelector} from 'react-redux'
+import CustomTextfield from "../../../components/CustomTextfield";
+import CustomButton from "../../../components/CustomButton";
+import { processLogin } from "../userSlice";
+import { toast } from "react-toastify";
+import { unwrapResult } from "@reduxjs/toolkit";
+
+const initialValue = {
+  email: '',
+  passWord: '',
+}
+
+const schema = yup.object({
+  email: yup.string().required("Required"),
+  passWord: yup.string().required("Required"),
+}).required()
+
+
 
 function Login() {
-  return <div>Login</div>;
+
+  const dispatch = useDispatch()
+  const navigate = useNavigate()
+  const user = useSelector(state => state.user.userInfo)
+  console.log("user: ", user)
+
+  const { control, handleSubmit, formState: { errors } } = useForm({
+    defaultValues: initialValue,
+    resolver: yupResolver(schema)
+  })
+
+  const onSubmitLogin = async (values) => {
+    try {      
+      const respone = await dispatch(processLogin(values))
+      const userInfo = unwrapResult(respone)
+      navigate("/")
+    } catch (error) {
+      toast.error("Login Failed!")     
+    }
+  }
+
+
+  return (
+    <Box sx={{ paddingTop: "50px" }}>
+      <Card
+        sx={{
+          margin: "0 auto",
+          maxWidth: 400,
+          padding: '8px 0px 6px'
+        }}
+      >
+        <Typography variant="h4" align="center" >Login</Typography>
+        <CardContent>
+          <form onSubmit={handleSubmit(onSubmitLogin)}>
```

Comment:
Please create a FormCompoent which receives an initial value as a prop. You should follow the presentation/container pattern.